vertigo-bot-executor/src/main/webapp/static/chatbot/js/chatbot.js

```diff
@@ -170,7 +171,15 @@ const chatbot = new Vue({
                     link.click();
                     document.body.removeChild(link);
                 }
-                chatbot.askBot(btn.payload, true);
+                chatbot.askBot(btn.payload, true, null,null);
+            },
+            fileUpload(btn, index) {
+                const file = document.getElementById('file_' + index).files[0];
+                const reader = new FileReader();
+                reader.readAsDataURL(file);

```

Comment:
Est ce qu'il y a un check sur la taille maximum du fichier par defaut ? et peut être sur le type de fichier qu'on accepte ? Il faudrait éviter de filer des .exe de 2Go au bot, je sais pas is vous en avez parlé déjà